lib/mapper.js

```diff
@@ -53,7 +54,6 @@ exports.track = function(track) {
   return extend(properties, {
     _p: track.userId() || track.sessionId(),
     _t: toUnixTimestamp(track.timestamp()),
-    _k: this.settings.apiKey,
```

Comment:
@f2prateek we're already appending that key in the through the `createBaseTrackProperties` function.